src/video.c

```diff
@@ -897,9 +893,10 @@ int video_alloc(struct video **vp, struct list *streaml,
 	tmr_init(&v->tmr);
 
 	err = stream_alloc(&v->strm, streaml, stream_prm,
-			   &cfg->avt, sdp_sess, MEDIA_VIDEO, label,
+			   &cfg->avt, sdp_sess, MEDIA_VIDEO, label, false, 0,
```

Comment:
the video stream also needs a jitter/reordering buffer ?

Reply:
Maybe yes. So I should drop the bool jbuf from stream_alloc()?